### PR TITLE
Create 0000-01-02-harahahk.md

### PR DESCRIPTION
**Pull request by theharshahk**
---
layout: slide
title: "Welcome to our second slide!"
---
welcome to git
Use the left arrow to go back!

### CODE REVIEW

**Comment by harahahk** [comment]
Great its working fine  👍

File: _posts/0000-01-02-harahahk.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (theharshahk) [inline _posts/0000-01-02-harahahk.md:1]: Great its working fine 👍

**Review by theharshahk — COMMENTED** [review]
Thank you

**Review by theharshahk — COMMENTED** [review]
👍👍Working great

File: _posts/0000-01-02-harahahk.md (hunk 1 above)
Comment (theharshahk) [inline _posts/0000-01-02-harahahk.md:1]: Nice !!

**Review by theharshahk — COMMENTED** [review]
Nice

**Review by theharshahk — COMMENTED** [review]
hi

**Pull request merged**